docs/concepts.rst

```diff
@@ -43,11 +43,11 @@ The :ref:`recoverWith` operation allows you to handle an event stream failure.
 Stream Failure vs. Application Errors
 `````````````````````````````````````
 
-Stream failures are different from *application errors*.  A stream failure indicates that an event stream *cannot* produce more events.  Application errors may or may not indicate an event stream failure.  For example, an event stream representing messages arriving on a WebSocket *fails* if the WebSocket is disconnected unexpectedly due to a wifi drop.  In contrast, an application error may occur in application logic that receives a WebSocket message and can't process because the message application state has changed.
+Stream failures are different from *application errors*.  A stream failure indicates that an event stream *cannot* produce more events.  Application errors may or may not indicate an event stream failure.  For example, an event stream representing messages arriving on a WebSocket *fails* if the WebSocket is disconnected unexpectedly due to a wifi drop.  In contrast, an application error may occur in application logic when it receives a WebSocket message that it can't process because the application state has changed.
```

Comment:
For some reason, GitHub isn't highlighting the diff here very well.  The last sentence is where the change is:

Old:
```
In contrast, an application error may occur in application logic that receives a WebSocket message and can't process because the message application state has changed.
```

New:
```
In contrast, an application error may occur in application logic when it receives a WebSocket message that it can't process because the application state has changed.
```

Reply:
Long lines.